### PR TITLE
[DOCS] Import value lists: renamed button, new flyout UI

### LINKED ISSUE #2262: [DOCS] "Upload value lists" button renamed "Import value lists"
## Description 

The "Upload value lists" button on the Rules page has been renamed "Import value lists". Confirmed that this changed between 8.3 and 8.4.

OLD:
image

NEW:
image

### PR DESCRIPTION
**Pull request by joepeeples**
Resolves #2262.

Previews:
- [Rule exceptions and value lists](https://security-docs_2268.docs-preview.app.elstc.co/guide/en/security/master/detections-ui-exceptions.html)
- See "Files changed" diff for other small edits

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
Looks great, thanks @joepeeples !

**Comment by deepikakeshav-qasource** [comment]
Hi @joepeeples,

We tested this PR and found 01 issue. 

- https://github.com/elastic/security-docs/issues/2310

We will Re-validate this PR once the above issue is fixed.

Thanks!

File: docs/detections/detections-ui-exceptions.asciidoc [hunk 1]
```diff
@@ -42,14 +44,14 @@ act as value delimiters.
...
-image::images/upload-lists-ui.png[]
+image::images/upload-lists-ui.png[Import value lists flyout,75%]
 
 . Select the list type (*Keywords*, *IP addresses*, *IP ranges*, or *Text*) from the *Type of value list* drop-down.
 . Drag or select the `csv` or `txt` file that contains the values.
-. Click *Upload list*.
+. Click *Import list*.
 
 NOTE: When the name of the file you are uploading already exists, the values in
 the new file are appended to the previously uploaded values.
```
Comment (joepeeples) [inline docs/detections/detections-ui-exceptions.asciidoc:57]:
```suggestion
NOTE: When the name of the file you are importing already exists, the values in
the new file are appended to the previously imported values.
```

**Comment by joepeeples** [comment]
> * [[BUG] `Upload` is displayed instead of `Import` under Note section #2310](https://github.com/elastic/security-docs/issues/2310)

Corrected in commit https://github.com/elastic/security-docs/pull/2268/commits/a7f5981bdee0a61d9acbe0e02fd3901d8f8bcb69. Thanks!

**Comment by deepikakeshav-qasource** [comment]
Hi @joepeeples,

We have reviewed the shared preview link and observed that all changes are present 🟢

image


Hence we are adding the "QA: Validated" label to it.

Thanks!!

**Review by benironside — APPROVED** [review]
LGTM

File: docs/detections/detections-ui-exceptions.asciidoc [hunk 2]
```diff
@@ -24,6 +24,8 @@ operators to define exceptions.
 IMPORTANT: Operators `is in list` and `is not in list` are not available for
 threshold and event correlation rules.
 
+TIP: You can also use value lists as the indicator match index when creating an indicator match rule. Refer to <<indicator-value-lists>> for more information.
```
Comment (joepeeples) [inline docs/detections/detections-ui-exceptions.asciidoc:27]:
```suggestion
TIP: You can also use value lists as the <<indicator-value-lists,indicator match index>> when creating an indicator match rule.
```

**Review by jmikell821 — APPROVED** [review]
Quick and easy -- LGTM! 👍

**Pull request merged**